src/CabanaPD_Solver.hpp

```diff
@@ -555,16 +551,8 @@ class SolverFracture
             // Update ghost particles.
             comm->gatherDisplacement();
 
-            // Compute/communicate LPS weighted volume (does nothing for PMB).
-            force->computeWeightedVolume( *particles, *neighbors, mu );
```

Comment:
Where is computeWeightedVolume now? I do not see it un updateForce.

Reply:
This is the next class - see line 603